### DISCUSSION TITLE
Preview file thumbnail

### DISCUSSION THREAD

**Discussion by oliviercperrier** [supabase/supabase#4554] [Feature Requests] [answered]
Hi,

I would like to recreate a similar dashboard as google drive and i was wondering if there is a not to complicated way to get file thumbnails using supabase storage?

Thank

**Comment by crasty01** [comment]
I'm not qualified too answer, but as I understand it, supabase only stores the file and does not create any thumbnails, so it's up to you. Either on backend, or frontend (but in frontend it makes no sense, because you are loading whole file either way...).

Feel free to correct me If I'm wrong ¯\_(ツ)_/¯

**Comment by inian** [comment]
Media transformations (including resizing images to produce thumbnails) is on the roadmap - probably Q2 or Q3 this year

**Reply by BhaskaranR** [reply]
@inian  great job on image resizing launch. any progress on thumbnails as well?

**Reply by inian** [reply]
resizing images to a smaller width should cover this right?

**Reply by oliviercperrier** [reply]
@inian For images, yes we could potentially do that for now, but if you check my comment, i was talking avec files (pdf, etc.) thumbnails.

**Comment by rajan-personal** [comment]
Why do signed video URLs need to be fully downloaded to preview the first frame, while other videos do not? What makes supabase signed URLs special?

**Comment by inian** [comment, marked as answer]
Image resizing is available - https://supabase.com/docs/guides/storage/serving/image-transformations#resizing
